app/components/layouts/DefaultLayout.tsx

```diff
@@ -45,7 +46,12 @@ function useHeroData({ Hero, heroHeader, heroSubheader }) {
 	}, [Hero, heroHeader, heroSubheader, matches]);
 }
 
-export function HeroHead({ Hero, heroHeader, heroSubheader, simple }) {
+export function HeroHead({
+	Hero,
+	heroHeader,
+	heroSubheader,
+	simple,
+}: DefaultLayoutProps) {
```

Comment:
I love the re-use of `DefaultLayoutProps`, but it includes a couple props that aren't being used in `HeroHead` (`showHero` and `children`). We might also run into issues in the future if we add more props to `DefaultLayout`.

Since this is a _subset_ of `DefaultLayoutProps`, we can use [`Pick`](https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys) and just grab the ones we want!

I think this should work:

```suggestion
}: Pick<DefaultLayoutProps, 'Hero' | 'heroHeader' | 'heroSubheader'| 'simple'>) {
```